src/lib.rs

```diff
@@ -56,6 +56,24 @@ where
     Err(())
 }
 
+#[cfg(target_env = "gnu")]
+mod glibc {
+    use std::path::Path;
```

Comment:
`Path` is already in scope in this file?

Reply:
I'm not against super imports, i just find them less explicit. In this case, I don't see the point since this is just one line.